src/Picker.tsx

```diff
@@ -0,0 +1,252 @@
+import React from "react";
+import {
+  Platform,
+  PickerIOS,
+  StyleSheet,
+  View,
+  TouchableOpacity,
+} from "react-native";
+import DialogAndroid, { OptionsPicker } from "react-native-dialogs";
+import Text from "./Text";
+import Modal from "./Modal";
+
+export interface Item {
+  value: string;
+  label: string;
+}
+
+export interface Props {
+  visible: boolean;
+  value: string;
+  items: Item[];
+  onDismiss: (value: string) => void;
+  onDone: (value: string) => void;
+  onCancel: (value: string) => void;
+  // This must be provided because if there is no done button,
+  // we cannot confirm select on Android.
+  doneButtonLabel?: string;
+  cancelButtonLabel?: string | null;
```

Comment:
What is the different bewteen cancelButtonLabel is undefined and is null?

Reply:
If it is undefined, it means use the default label "Cancel".
If it is null, it means do not show the cancel button at all.